lib/email/styles.rb

```diff
@@ -140,9 +140,12 @@ def onebox_styles
       style('.github-icon-container', 'float: left;')
       style('.github-icon-container *', 'fill: #646464; width: 40px; height: 40px;')
       style('.github-body-container', 'font-family: Consolas, Menlo, Monaco, "Lucida Console", "Liberation Mono", "DejaVu Sans Mono", "Bitstream Vera Sans Mono", "Courier New", monospace; margin-top: 1em !important;')
-      style('.github-body-container .excerpt', 'display: none;')
       style('.onebox-avatar-inline', ONEBOX_INLINE_AVATAR_STYLE)
 
+      @fragment.css('.github-body-container .excerpt').each do |el|
+        el.remove
+      end
+
```

Comment:
I think you can call `remove` (or `delete`) directly

```suggestion
      @fragment.css('.github-body-container .excerpt').remove
```

Reply:
Nice.